apps/src/templates/LargeChevronLink.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {Component, PropTypes} from 'react';
+import FontAwesome from './FontAwesome';
+import color from "../util/color";
+import Radium from 'radium';
+
+const styles = {
+  link: {
+    textDecoration: 'none',
+  },
+  linkBox: {
+    marginTop: 10,
+    marginBottom: 10,
+  },
+  linkText: {
+    fontSize: 20,
+    lineHeight: '22px',
+    fontFamily: 'Gotham 5r',
+    color: color.teal,
+    clear: 'both',
+    display: 'inline',
+    fontWeight: 'bold',
+    marginLeft: 15,
+    marginRight: 15,
+  },
+  chevron: {
+    display: 'inline',
+    color: color.teal,
+    fontSize: 20,
+    fontWeight: 'bold',
+  },
+  left: {
+    float: 'left',
+  },
+  right: {
+    float: 'right',
+  },
+};
+
+class LargeChevronLink extends Component {
+  static propTypes = {
+    linkText: PropTypes.string.isRequired,
+    link: PropTypes.string.isRequired,
+    isRtl: PropTypes.bool.isRequired,
+  };
+
+  render() {
+    const { link, linkText, isRtl }= this.props;
+    const localeStyle = isRtl ? styles.right : styles.left;
+    const icon = isRtl ? "chevron-right" : "chevron-left";
+
+    return (
+      <div style={[styles.linkBox, localeStyle]}>
```

Comment:
Have we settled on when to use arrays and when to use es6 object spread (`...`)?

Reply:
There are many examples of both in the codebase. I don't know of a reason to feel strongly in favor of one or the other.